terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -218,6 +213,8 @@ locals {
       "fastq_list_rows": null,
       "output_file_prefix": null,
       "output_directory": null,
+      "enable_map_align_output": true,
+      "enable_duplicate_marking": false,
```

Comment:
Right. That's where Pete mentioned, for transcriptome, this is false. Ok, gotcha. Also, understood that these will be static flag at input template (ditto above).

Reply:
Yep